cmd/balance-worker/main.go

```diff
@@ -87,7 +87,9 @@ func main() {
 
 	err = conf.Validate()
 	if err != nil {
-		panic(err)
+		println("configuration error:")
+		println(err.Error())
+		os.Exit(1)
```

Comment:
I wonder how this should be reported in logs. If there are more than one errors, the log line is going to be loooong.

Reply:
@chrisgacsal any thoughts on this?

Reply:
How about unwrapping the error and print each error separately?

Reply:
It should already print each error on a separate line.

I think I'm gonna leave it like this for now, we can always fix it later.